src/coprocessor/endpoint.rs

```diff
@@ -273,6 +278,9 @@ impl Display for Task {
 enum CopRequest {
     DAG(DAGRequest),
     Analyze(AnalyzeReq),
+    Get(GetRequest),
```

Comment:
A bit of ugly here. Get, BatchGet, Scan is not the same level with DAG.

Reply:
+1

Reply:
How about KV(KvRequest)?

Reply:
It is expected to be refined in a later version.

Reply:
Here I think you just need a read thread pool to handle all Scheduler and Coprocessor read, you can dispatch the Scheduler and Coprocessor in gRPC service directly.